app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+  def self.comments_counter(parameters)
+    post = Post.find_by(id: parameters)
+    post.comments_counter = post.Update(comments_counter: comments_counter + 1)
+  end
```

Comment:
- [ ] Using the self keyword when creating a method is redundant. Kindly remove the `self.` keyword and do the same for `likes_counter` and `post_counter` methods.